solara/server/templates/solara.html.j2

```diff
@@ -38,9 +36,22 @@
         "kernelId": "1234"
     }
     </script>
-    <style>
+    <style class="solara-template-css">
         {% include "loader-"~theme.loader~".css" %}
     </style>
+    <style id="pre-render-theme">
+    .theme--light .v-sheet {
+        background-color: #fff;
+        border-color: #fff;
+        color: rgba(0,0,0,.87)
+    }
+
+    .theme--dark .v-sheet {
+        background-color: #1e1e1e;
+        border-color: #1e1e1e;
+        color: #fff
+    }
+</style>
```

Comment:
@maartenbreddels what do you think, should these options be configurable?

Reply:
Can you explain why it is there in a comment, and why we remove it later on?
If I think I understand (first render) than this is fine for now I think.